pkg/vpcmodel/nodesExplainability.go

```diff
@@ -297,6 +315,92 @@ func computeActualRules(rulesLayer *rulesInLayers, filtersExternal map[string]bo
 	return &actualRules, directionEnabled
 }
 
+// computes combined list of rules, both deny and allow
+func (details *rulesAndConnDetails) computeCombinedActualRules() {
+	for _, singleSrcDstDetails := range *details {
+		actualRulesIngress := mergeAllowDeny(singleSrcDstDetails.actualAllowRules.ingressRules,
+			singleSrcDstDetails.actualDenyRules.ingressRules)
+		actualRulesEgress := mergeAllowDeny(singleSrcDstDetails.actualAllowRules.egressRules,
+			singleSrcDstDetails.actualDenyRules.egressRules)
+		actualRules := &rulesConnection{actualRulesIngress, actualRulesEgress}
+		singleSrcDstDetails.actualMergedRules = actualRules
+	}
+}
+
+// merges two rulesInLayers - for merging deny and allow for ingress and egress
+func mergeAllowDeny(allow, deny rulesInLayers) rulesInLayers {
+	allowDenyMerged := rulesInLayers{}
+	for _, layer := range filterLayers {
+		allowForLayer, ok1 := allow[layer]
+		denyForLayer, ok2 := deny[layer]
+		switch {
+		case ok1 && ok2:
+		case ok1: // layer relevant only for deny
```

Comment:
```suggestion
		case ok1 && ok2:
		  // do nothing (merge will be right after the switch)
		case ok1: // layer relevant only for deny
```

Reply:
added manually (it won't let me commit the change, probably due to other changes)